README.md

```diff
@@ -1,8 +1,27 @@
 # noir-ethereum-history-api
 
+noir-ethereum-history-api is an API that lets you prove and verify account/storage states of any block in a blockchain (more about zero-knowledge proofs: [Zero knowledge - Wikipedia](https://en.wikipedia.org/wiki/Zero_knowledge#:~:text=Zero%2Dknowledge%20proof%2C%20a%20concept,the%20veracity%20of%20the%20statement)). It is currently in a development stage.
```

Comment:
- use impersonal form
- ... historical accounts balances and smart contract storage ... veryfi claims on Ethereum
- extract stage to separate paragraph 
- separate paragraph on verifying onchain only last 256 blocks